src/winetricks

```diff
@@ -13022,6 +13023,7 @@ load_vcrun2019()
             # 2021/06/06: a1592d3da2b27230c087a3b069409c1e82c2664b0d4c3b511701624702b2e2a3
             # 2021/08/26: 003063723b2131da23f40e2063fb79867bae275f7b5c099dbd1792e25845872b
             # 2021/10/23: 9b9dd72c27ab1db081de56bb7b73bee9a00f60d14ed8e6fde45dab3e619b5f04
+            # 2022/01/18: 9b9dd72c27ab1db081de56bb7b73bee9a00f60d14ed8e6fde45dab3e619b5f04
```

Comment:
This should be 296f96cd102250636bcd23ab6e6cf70935337b1bbb3507fe8521d8d9cfaa932f (and the value should be changed in the w_download call below)